cli/tests/e2e/snapshots/test_output/test_output_highlighting__force_color_and_no_color/results.txt

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
This blank line is where the rule parse progress bar would go in interactive mode.